mac/KeymanEngine4Mac/KeymanEngine4Mac/CoreWrapper/CoreKeyOutput.m

```diff
@@ -32,9 +32,10 @@ -(instancetype)init:(NSUInteger)codePointsToDelete textToDelete:(NSString*)delet
 
 -(NSString *)description
 {
-  NSData* data = [self.textToInsert dataUsingEncoding:NSUTF16LittleEndianStringEncoding];
-  
-  return [[NSString alloc] initWithFormat: @"codePointsToDeleteBeforeInsert: %li, textToInsert: '%@', optionsToPersist: %@, alert: %d, emitKeystroke: %d, capsLockState: %d ", self.codePointsToDeleteBeforeInsert, data, self.optionsToPersist, self.alert, self.emitKeystroke, self.capsLockState];
+  NSData* deleteData = [self.textToDelete dataUsingEncoding:NSUTF16LittleEndianStringEncoding];
```

Comment:
The description method is called to log the CoreKeyOutput state from CoreWrapper processMacVirtualKey. Both `textToInsert` and `textToDelete` can be null, but if they are, then `dataUsingEncoding` returns null, and the log statement will say  `textToDelete: '(null)' [data: '(null)']` 

Because this is being logged with a debug statement, this description method will only be executed when the user is streaming debug logs.